src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/IndividualRepo.java

```diff
@@ -459,6 +487,33 @@ public SolrQuery query() {
             return this.query;
         }
 
+        public JsonQueryRequest jsonQuery(List<String> ids) {
+            final ModifiableSolrParams params = new ModifiableSolrParams();
+            params.set("q.op", defaultOperator);
+
+            JsonQueryRequest request = new JsonQueryRequest(params)
+                .setQuery(DEFAULT_QUERY)
+                .setLimit(this.query.getRows());
+
+            String termFilter = String.format("{!terms f=id}:%s", String.join(",", ids)); 
```

Comment:
This was a good find on stack overflow. Term query instead of increasing Boolean operators.